### PR TITLE
doc: Add missing documentation

### PR DESCRIPTION
**Pull request by zekth**
Added links to module documentation.
Added fs documentation.

Mention to @axetroy , could you provide an explanation for the options of `writeJson` module options please?

### CODE REVIEW

**Comment by axetroy** [comment]
@zekth
The options is same with `JSON.stringify(value[, replacer[, space]])`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#Parameters

File: fs/README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+Reads a JSON file and then parses it into an object
+
+```ts
+import { readJson, readJsonSync } from "https://deno.land/std/fs/mod.ts";
+
+const f = await readJson("./foo.json");
+const foo = readJsonSync("./foo.json");
+```
+
+### utils
```
Comment (axetroy) [inline fs/README.md]: I think `utils` modules should not expose by the user.

currently only used in internal modules, may change d in the future.

File: fs/README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+# fs
+
+fs module is made to provide helpers to manipulate the filesystem.
+
+## Usage
+
+All the following modules are exposed in `mod.ts`
+
+### empty_dir
```
Comment (axetroy) [inline fs/README.md]:
```suggestion
### emptyDir
```

Comment (axetroy) [reply fs/README.md]: Shouldn't it be the same as the method?

File: fs/README.md (hunk 1 above)
Comment (zekth) [reply fs/README.md]: Right it's not exposed in mod.ts

File: fs/README.md (hunk 2 above)
Comment (zekth) [reply fs/README.md]: My bad, was looking at filename not exposed function name.

**Comment by zekth** [comment]
> LGTM - thanks
> 
> I plan on doing some automated documentation at some point, but for the time being this is helpful.

Once denoland/deno#2089 will land i have a draft for TypeDoc autodocumentation.

File: fs/README.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+If the directory structure does not exist, it is created. Like mkdir -p.
+
+```ts
+import { ensureDir, ensureDirSync } from "https://deno.land/std/fs/mod.ts";
+
+ensureDir("./bar"); // returns a promise
+ensureDirSync("./ensureDirSync"); // void
+```
+
+### ensure_file
```
Comment (axetroy) [inline fs/README.md]: here should be `ensureFile`
``

**Pull request merged**